flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/copy/HiveParserSqlFunctionConverter.java

```diff
@@ -415,24 +425,73 @@ public static SqlOperator getCalciteFn(
                                 OperandTypes.PLUS_OPERATOR);
                 break;
             default:
-                calciteOp = HIVE_TO_CALCITE.get(hiveUdfName);
-                if (null == calciteOp) {
-                    calciteOp =
-                            new CalciteSqlFn(
-                                    uInf.udfName,
-                                    uInf.identifier,
-                                    SqlKind.OTHER_FUNCTION,
-                                    uInf.returnTypeInference,
-                                    uInf.operandTypeInference,
-                                    uInf.operandTypeChecker,
-                                    SqlFunctionCategory.USER_DEFINED_FUNCTION,
-                                    deterministic);
+                // some functions should be handled as Hive UDF for
+                // the Hive specific logic
+                if (shouldHandleAsHiveUDF(uInf) && canHandledAsHiveUDF(uInf, functionConverter)) {
+                    calciteOp = asCalciteSqlFn(uInf, deterministic);
+                } else {
+                    calciteOp = HIVE_TO_CALCITE.get(hiveUdfName);
```

Comment:
A few minor suggestions:

1. The comments should be started with a capital letter.
2. shouldHandleAsHiveUDF->shouldHandledAsHiveUDF to be consistent with canHandledAsHiveUDF

Reply:
But about the comments, I think it's fine to not to change it for some some comments in this file are started with a capital letter and some are not.